pkg/aggregator/check_sampler.go

```diff
@@ -37,7 +52,88 @@ func (cs *CheckSampler) addSample(metricSample *metrics.MetricSample) {
 	}
 }
 
-func (cs *CheckSampler) commit(timestamp float64) {
+func (cs *CheckSampler) newSketchSeries(ck ckey.ContextKey, points []metrics.SketchPoint) metrics.SketchSeries {
+	ctx := cs.contextResolver.contextsByKey[ck]
+	ss := metrics.SketchSeries{
+		Name: ctx.Name,
+		Tags: ctx.Tags,
+		Host: ctx.Host,
+		// Interval: TODO: investigate
```

Comment:
@jbarciauskas I'm not sure how to handle this field in the context of checks?

Reply:
Do checks run at a standard interval? I think it's only important for counts

Reply:
@jbarciauskas they usually do (15 sec), but a custom check interval can be defined per check